src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SPLITTER = ",";
+    private static final int OPERATION_NAME_INDEX = 0;
+    private static final int AMMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeStatistic(toFileName, computeStatistic(readStatistic(fromFileName)));
+    }
+
+    public String[][] readStatistic(String fromFileName) {
```

Comment:
If yes, so we can come up with a better name
```suggestion
    public String[][] readFromFile(String fromFileName) {
```